src/ft/mod.rs

```diff
@@ -173,7 +173,109 @@ impl Rasterize for FreeTypeRasterizer {
     }
 
     fn get_glyph(&mut self, glyph_key: GlyphKey) -> Result<RasterizedGlyph, Error> {
-        self.get_rendered_glyph(glyph_key)
+        // Render a normal character if it's not a cursor.
+        let font_key = self.face_for_glyph(glyph_key);
+        let face = &self.loader.faces[&font_key];
+        let index = face.ft_face.get_char_index(glyph_key.character as usize);
+        let pixelsize = face
+            .non_scalable
+            .unwrap_or_else(|| glyph_key.size.as_f32_pts() * self.device_pixel_ratio * 96. / 72.);
+
+        if !face.colored {
+            face.ft_face.set_char_size(to_freetype_26_6(pixelsize), 0, 0, 0)?;
+        }
+
+        unsafe {
+            let ft_lib = self.loader.library.raw();
+            freetype::ffi::FT_Library_SetLcdFilter(ft_lib, face.lcd_filter);
+        }
+
+        face.ft_face.load_glyph(index, face.load_flags)?;
+
+        let glyph = face.ft_face.glyph();
+
+        // Generate synthetic bold.
+        if face.embolden {
+            unsafe {
+                freetype_sys::FT_GlyphSlot_Embolden(glyph.raw()
+                    as *const freetype_sys::FT_GlyphSlotRec
+                    as *mut freetype_sys::FT_GlyphSlotRec);
+            }
+        }
+
+        // Transform glyphs with the matrix from Fontconfig. Primarily used to generate italics.
```

Comment:
This comment is now off. Probably should move closer to matrix and not to advance stuff?

Also, have you verified how advance work with emoji fonts and bitmaps? I'm worried that they could a little bit special here, so you'd need to scale advance for them.